src/display-buffer.coffee

```diff
@@ -1174,7 +1174,7 @@ class DisplayBuffer extends Model
 
   destroyed: ->
     fold.destroy() for markerId, fold of @foldsByMarkerId
-    marker.disposables.dispose() for id, marker of @markers
```

Comment:
Nevermind, look like it was originally added by me quite a long time ago in https://github.com/atom/atom/commit/1764677b7d039fbcbd9febef550703b2b9a1c04c to fix some sort of regression